cassandra/README.md

```diff
@@ -43,195 +43,171 @@ To complete this tutorial, you should already have a basic familiarity with [Pod
-cassandra-0   1/1       Running   0          10m
-cassandra-1   1/1       Running   0          9m
-cassandra-2   1/1       Running   0          8m
-```
+       NAME          READY     STATUS    RESTARTS   AGE
+       cassandra-0   1/1       Running   0          10m
+       cassandra-1   1/1       Running   0          9m
+       cassandra-2   1/1       Running   0          8m
 
-{:start="3"}
 3. Run the Cassandra utility nodetool to display the status of the ring.
 
-```
-kubectl exec cassandra-0 -- nodetool status
-```
-
-```console
-Datacenter: DC1-K8Demo
-======================
-Status=Up/Down
-|/ State=Normal/Leaving/Joining/Moving
---  Address   Load       Tokens       Owns (effective)  Host ID                               Rack
-UN  10.4.2.4  65.26 KiB  32           63.7%             a9d27f81-6783-461d-8583-87de2589133e  Rack1-K8Demo
-UN  10.4.0.4  102.04 KiB  32           66.7%             5559a58c-8b03-47ad-bc32-c621708dc2e4  Rack1-K8Demo
-UN  10.4.1.4  83.06 KiB  32           69.6%             9dce943c-581d-4c0e-9543-f519969cc805  Rack1-K8Demo
-```
+       kubectl exec cassandra-0 -- nodetool status
+
+   The response is:
+
+       Datacenter: DC1-K8Demo
+       ======================
+       Status=Up/Down
+       |/ State=Normal/Leaving/Joining/Moving
+       --  Address     Load       Tokens       Owns (effective)  Host ID                               Rack
+       UN  172.17.0.5  83.57 KiB  32           74.0%             e2dd09e6-d9d3-477e-96c5-45094c08db0f  Rack1-K8Demo
+       UN  172.17.0.4  101.04 KiB  32           58.8%             f89d6835-3a42-4419-92b3-0e62cae1479c  Rack1-K8Demo
+       UN  172.17.0.6  84.74 KiB  32           67.1%             a6a1e8c2-3dc5-4417-b1a0-26507af2aaad  Rack1-K8Demo
 
 ## Modifying the Cassandra StatefulSet
 Use `kubectl edit` to modify the size of of a Cassandra StatefulSet. 
 
 1. Run the following command:
 
-```shell
-kubectl edit statefulset cassandra
-```
+       kubectl edit statefulset cassandra
+
    This command opens an editor in your terminal. The line you need to change is `Replicas`.
    
    **Note:** The following sample is an excerpt of the StatefulSet file.
-
-```console
-# Please edit the object below. Lines beginning with a '#' will be ignored,
-# and an empty file will abort the edit. If an error occurs while saving this file will be
-# reopened with the relevant failures.
-#
-apiVersion: apps/v1beta1
-kind: StatefulSet
-metadata:
-  creationTimestamp: 2016-08-13T18:40:58Z
-  generation: 1
-  labels:
-    app: cassandra
-  name: cassandra
-  namespace: default
-  resourceVersion: "323"
-  selfLink: /apis/apps/v1beta1/namespaces/default/statefulsets/cassandra
-  uid: 7a219483-6185-11e6-a910-42010a8a0fc0
-spec:
-  replicas: 3
-```
-
-{:start="2"}
-2. Increase the number of replicas to 4, and then save the manifest. 
-
-   The StatefulSet now contains 4 pods.
+   {: .note}
+   
+       # Please edit the object below. Lines beginning with a '#' will be ignored,
```

Comment:
can you use 3-backticks syntax instead of indentation for such long snippets?

Also use <pre>```yaml</pre> 

syntax to provide syntax highlighting in such snippets.